tests/PHPStan/Rules/Methods/data/call-static-method-mixed.php

```diff
@@ -0,0 +1,171 @@
+<?php // lint >= 8.0
+
+namespace CallStaticMethodMixed;
+
+class Foo
+{
+
+	/**
+	 * @param mixed $explicit
+	 */
+	public function doFoo(
+		$implicit,
+		$explicit
+	): void
+	{
+		$implicit::foo();
+		$explicit::foo();
+	}
+
+	/**
+	 * @template T
+	 * @param T $t
+	 */
+	public function doBar($t): void
+	{
+		$t::foo();
+	}
+
+}
+
+class Bar
+{
+
+	/**
+	 * @param mixed $explicit
+	 */
+	public function doFoo(
+		$implicit,
+		$explicit
+	): void
+	{
+		self::doBar($implicit);
+		self::doBar($explicit);
+
+		self::acceptImplicitMixed($implicit);
+		self::acceptImplicitMixed($explicit);
+
+		self::acceptExplicitMixed($implicit);
+		self::acceptExplicitMixed($explicit);
+
+		self::acceptVariadicArguments(...$implicit);
+		self::acceptVariadicArguments(...$explicit);
+	}
+
+	public static function doBar(int $i): void
+	{
+
+	}
+
+	public static function acceptImplicitMixed($mixed): void
+	{
+
+	}
+
+	public static function acceptExplicitMixed(mixed $mixed): void
+	{
+
+	}
+
+	public static function acceptVariadicArguments(mixed... $args): void
+	{
+
+	}
+
+	/**
+	 * @template T
+	 * @param T $t
+	 */
+	public function doLorem($t): void
+	{
+		self::doBar($t);
+		self::acceptImplicitMixed($t);
+		self::acceptExplicitMixed($t);
+		self::acceptVariadicArguments(...$t);
+	}
+
+}
+
+class CallableMixed
+{
+
+	/**
+	 * @param callable(mixed): void $cb
```

Comment:
Unfortunately, I wasn't able to figure out how to make a callable with implicit mixed parameter. But if it's impossible to make one, then it doesn't matter how it behaves and so there's no need to test it.